pages/_template.js

```diff
@@ -88,14 +96,26 @@ class Template extends Component {
     return _.result(_.find(helmet.metaTags, [ 'name', 'mixpanelTitle' ]), 'content');
   };
 
+  closeBetaBanner() {
+    localStorage.setItem('closeBetaBanner', true);
+    this.setState({ closeBetaBanner: true });
+  };
+
+  onApplyBeta() {
+    console.log('Apply for beta');
```

Comment:
that's a place holder, I am working on the referal page - the one that opens after onApplyBeta(), on a different branch.